packages/models/src/crypto-assets/bitcoin/inscription.model.ts

```diff
@@ -156,3 +157,82 @@ export function createInscription(inscription: RawInscription): Inscription {
     }),
   });
 }
+
+export function createBestinSlotInscription(
+  inscription: BestinslotInscriptionResponse
+): Inscription {
+  const id = inscription.inscription_id;
+  const contentSrc = `${HIRO_INSCRIPTIONS_API_URL}/${id}/content`;
+  const iframeSrc = `https://ordinals.com/preview/${id}`;
+  const preview = `https://ordinals.hiro.so/inscription/${id}`;
+  const title = `Inscription ${inscription.inscription_number}`;
+  const [txid, output, offset] = inscription.satpoint.split(':');
+
+  const sharedInfo = {
+    id: inscription.inscription_id,
+    number: inscription.inscription_number,
+    output,
+    txid,
+    offset,
+    address: inscription.owner_wallet_address,
+    preview,
+    title,
+    genesisBlockHeight: inscription.genesis_height,
+
+    // TODO: currently no data from BestinSlot API
+    genesisBlockHash: '0',
+    genesisTimestamp: 0,
+    value: '0',
```

Comment:
wonder if we need this info? 🤔 it doesn't come from bis api, should we ask to fix it?